.gitignore

```diff
@@ -60,4 +60,18 @@ local.properties
 
 
 ## VSCODE ***
-/.vscode/
\ No newline at end of file
+/.vscode/
+
+##############################
+## Workflow Service SDK Generated files
+##############################
+
+workflow-service-sdk/src/
```

Comment:
I don't think we want this :-) 
It will ignore any changes to the project.
Perhaps there is an option to specify the output directory for the generated resource?

Reply:
I'm already using the [option](https://github.com/redhat-developer/parodos/blob/de8579b246d6991d08f63621cc41db09a266f298/workflow-service-sdk/pom.xml#L99) to specify the output directory. 

So my doubt here is: do we want to commit/push any change in the generated code? I thought we are interested in the library jar, that's why I added this to the `.gitignore`.